#!/Users/a/Downloads/DataScience_World/LLM_project/geminiLLMapp/venv/bin/python3.12
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
